lib/stack/src/lib.rs

```diff
@@ -63,22 +63,22 @@ pub mod shared {
     pub struct Stack<V: Clone>(V);
 
     #[derive(Debug)]
-    pub enum Error {}
+    pub enum Never {}
 
     impl<T, V: Clone> super::Stack<T> for Stack<V> {
         type Value = V;
-        type Error = Error;
+        type Error = Never;
```

Comment:
TIOLI: It might be worth adding a comment that this should be replaced with `!` when it becomes stable?

Reply:
:-1: i don't find these comments to be useful